reference_definitions_schemas/new_schema_refs.json

```diff
@@ -34,6 +34,10 @@
       "description": "Chromatin amount",
       "type": "number"
     },
+    "controlType": {
+      "description": "Control samples suitable for normalization and batch correction",
```

Comment:
I don't know how to interpret this definition and I'm further confused by numbers as the allowable type.

Reply:
@kelshmo I've corrected the type to string.

I don't know how to interpret the definition either, it was copied directly from the AMP-AD TMTquantitation assay.  @amapeters , @karawoo , can you clarify?

Reply:
@amapeters please comment on specific controls used e.g. GIS.